test/unit/math/prim/mat/meta/VectorViewMvt_test.cpp

```diff
@@ -0,0 +1,51 @@
+#include <stan/math/prim/mat.hpp>
+#include <gtest/gtest.h>
+#include <iostream>
+#include <stdexcept>
+#include <vector>
+
+// Just test the use case where a single matrix is passed in to VectorViewMvt.
+// The case where a std::vector of matrices is passed in is broken and will be refactored.
+
+TEST(MetaTraits, VectorViewMvt_matrix_double) {
+  using stan::VectorViewMvt;
+  using Eigen::Matrix;
+  using Eigen::Dynamic;
+
+  Matrix<double,Dynamic,1> a(10);
+  for (size_t n = 0; n < 10; ++n)
+    a[n] = n;
+  VectorViewMvt<Matrix<double,Dynamic,1> > av(a);
```

Comment:
It seems like I should replace these with the `VectorXd` and `RowVectorXd` for clarity then, right? Will do. Copied that style from the existing tests for `VectorView` but am happy to improve upon it.